### PR TITLE
Document changing load balancer address

### PR DESCRIPTION
**Pull request by areed**

### CODE REVIEW

File: src/markdown-pages/install-with-kurl/index.md [hunk 1]
```diff
@@ -23,13 +23,27 @@ HA installs will prompt and wait for an optional load balancer address to be pro
...
 
-```
+```bash
 curl https://kurl.sh/latest | sudo bash -s ha
 ```
 
+If the cluster has remote nodes, the install script will print out a command that must be run on every node to update clients to use the new load balancer address:
+
+```bash
+curl https://kurl.sh/latest/tasks.sh | sudo bash -s set-kubeconfig-server https://loadbalancer.internal:6443
```
Comment (laverya) [inline src/markdown-pages/install-with-kurl/index.md:35]: Will it always be this literal command? Or will the URL change based on the LB address?

Comment (areed) [reply src/markdown-pages/install-with-kurl/index.md:35]: It will vary based on the LB address

Comment (laverya) [reply src/markdown-pages/install-with-kurl/index.md:35]: we should have it more clearly be variable in these docs then IMO

Comment (laverya) [reply src/markdown-pages/install-with-kurl/index.md:35]: Maybe `https://k8slb.somebigbank.com:6443` @areed ?

Comment (areed) [reply src/markdown-pages/install-with-kurl/index.md:35]: Sounds good

**Review by laverya — APPROVED** [review]

**Pull request merged**